roles/lib_openshift/library/oc_objectvalidator.py

```diff
@@ -1380,7 +1380,14 @@ def _is_invalid_namespace(namespace):
                 all_invalid[invalid_msg] = invalid
 
         if failed:
-            return {'failed': True, 'msg': 'All objects are not valid.', 'state': 'list', 'results': all_invalid}
+            return {
+                'failed': True,
+                'msg': "All objects are not valid.  If you are a supported customer please contact \
+                        Red Hat Support providing the complete output above. If you are not a customer \
+                        please contact users@lists.openshift.redhat.com for assistance.",
```

Comment:
That's definitely readable but it still seems to get formatted by ansible. Perhaps ansible is doing something with the string. Anyway, it's a failure condition, we can deal with slightly imperfect formatting. Thanks.